src/functions/update-curation/handler.ts

```diff
@@ -42,13 +39,13 @@ const updateCuration: ValidatedEventAPIGatewayProxyEvent<typeof schema> = async
     }
   }
 
-  const hasDocumentAttributes = !!(title && description && links && language);
-  if (curationType === CurationType.CUSTOM && !hasDocumentAttributes) {
+  if (curationType === CurationType.CUSTOM && !document) {
     return {
       statusCode: 400,
       body: "Bad request, missing document values"
     };
   }
+  const { title, description, links, language } = document!;
```

Comment:
Same here, no need to access these in the top level of the function, as they are only needed in the specific parts inside conditions.